bindings/node/lib/clientEncryption.js

```diff
@@ -196,12 +196,12 @@ module.exports = function(modules) {
      */
 
     /**
-     * Explicitly encrypt a provided value. Note that either `options.dataKey` or `options.keyAltName` must
-     * be specified. Specifying both `options.dataKey` and `options.keyAltName` is considered an error.
+     * Explicitly encrypt a provided value. Note that either `options.keyId` or `options.keyAltName` must
+     * be specified. Specifying both `options.keyId` and `options.keyAltName` is considered an error.
      *
      * @param {*} value The value that you wish to serialize. Must be of a type that can be serialized into BSON
      * @param {object} options
-     * @param {ClientEncryption~dataKey} [options.dataKey] The Binary dataKey to use for encryption
+     * @param {ClientEncryption~dataKey} [options.keyId] The Binary dataKey to use for encryption
```

Comment:
should this become `ClientEncryption~keyId` instead of `dataKey`? Also, what is the impetus for the change from `dataKey` => `keyId`?

Reply:
The motivation was originally b/c the documentation incorrectly identified the field as `dataKey` instead of `keyId`. So at a bare minimum we need to properly document that the field is called `keyId`.

I'd be on board with renaming `ClientEncryption~dataKey` to `ClientEncryption~dataKeyId`

Reply:
I think it looks strange to have `clientEncryption.createDataKey` return something called a `keyId`. I would expect it to return a "data key"

Reply:
It creates a `dataKey`, and inserts that `dataKey` into the key vault namespace, and then returns the `_id` of that `dataKey`. It is explicitly not the `dataKey` b/c you could not insert that into the keyVaultNamespace and have it work.

Reply:
ah right right. Okay, this LGTM as long as you make everything consistent (`dataKeyId` sounds good to me too).